### DISCUSSION TITLE
Persistent connections are disabled at webserver and can't be enabled

### DISCUSSION THREAD

**Discussion by horus112** [magicbug/Cloudlog#1655] [Q&A] [answered]
After installation I got the following error message:

"A PHP Error was encountered
Severity: Warning
Message: mysqli::real_connect(): Persistent connections are disabled. Downgrading to normal
Filename: mysqli/mysqli_driver.php
Line Number: 201"

How can I switch permanent to default connection?

**Comment by magicbug** [comment, marked as answer]
edit the /applications/config/database.php file

**Reply by horus112** [reply]
Thank you! That was easy.
